gcloud/bigquery/dataset.py

```diff
@@ -211,8 +212,10 @@ def _set_properties(self, api_response):
         """
         self._properties.clear()
         cleaned = api_response.copy()
-        cleaned['creationTime'] = float(cleaned['creationTime'])
-        cleaned['lastModifiedTime'] = float(cleaned['lastModifiedTime'])
+        if 'creationTime' in cleaned:
+            cleaned['creationTime'] = float(cleaned['creationTime'])
+        if 'lastModifiedTime' in cleaned:
+            cleaned['lastModifiedTime'] = float(cleaned['lastModifiedTime'])
```

Comment:
Sorry for just noticing, but the [discovery doc][1] indicates that `lastModifiedTime` and `creationTime` are `int64`. Why the conversion to `float` instead? (Ditto below.)

[1]: https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest